packages/mui-codemod/src/deprecations/backdrop-props/test-cases/actual.js

```diff
@@ -0,0 +1,27 @@
+import Backdrop from '@mui/material/Backdrop';
+import { Backdrop as MyBackdrop } from '@mui/material';
+
+<Backdrop TransitionComponent={CustomTransition} transitionDuration={100} />;
```

Comment:
The `transitionDuration` ones should be removed here as well as the modified version on the expected cases